spec/dummy/app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,7 @@
 
 // Third Party Libraries
 @import 'bulma';
+@import '../../../../../app/components/bali/calendar/index';
```

Comment:
Please try is using `~` works on this.
```suggestion
@import '~/app/components/bali/calendar/index';
```

Reply:
I thought this method was used when importing CSS from a node_module. I think here it's different.

Reply:
but it's worth a try

Reply:
it didn't work 🤷‍♂️

Reply:
<img width="867" alt="Captura de Pantalla 2022-06-10 a la(s) 12 23 01" src="https://user-images.githubusercontent.com/37639003/173135882-3eaa3a5c-46dc-4cef-a43f-a02e6b5ce04f.png">